docs/app/Examples/elements/Icon/IconSet/IconSetExamples.js

```diff
@@ -1,16 +1,21 @@
+import _ from 'lodash'
 import React, { Component } from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+import { iconCats as categories } from 'src/utils/semanticutils'
 
 export default class IconSetExamples extends Component {
   render() {
     return (
       <ExampleSection title='Icon Set'>
-        <ComponentExample
-          title='Web Content'
-          description='Icons can represent types of content found on websites'
-          examplePath='elements/Icon/IconSet/IconWebContentExample'
-        />
+        {_.map(categories, category => (
+          <ComponentExample
+            title={category.name}
+            description='Icons can represent types of content found on websites'
+            examplePath='elements/Icon/IconSet/IconCategoryExample'
+            category={category}
+          />
+        ))}
       </ExampleSection>
```

Comment:
Pulled your branch and I see the conundrum here.  I would prefer we keep very real world example code for the users to see.  Being that all the other great examples you've added do that, I have an idea.

What if we just show a single row of hard coded `<Icon />`s and include a Message that points the users to the SUI icon docs for a complete list of supported icons?  If we get complaints, or better yet a PR to improve it, we'll merge it 😸